shader/CMakeLists.txt

```diff
@@ -196,6 +196,9 @@ add_shader(tonemapping.vert          copy.vert -DHAS_UV)
 add_shader(tonemapping.frag          lighting/tonemapping.frag)
 add_shader(tonemapping_up.frag       lighting/tonemapping.frag -DUPSCALE)
 
+add_shader(tonemapping.comp          lighting/tonemapping.frag -S comp -DCOMPUTE)
+add_shader(tonemapping_up.comp       lighting/tonemapping.frag -S comp -DCOMPUTE -DUPSCALE)
```

Comment:
Does it make sense to have need anti-aliasing + upscale together?

Reply:
In FSR1 documentation there's a note that it's better to use antialiased input to FSR1 and we have the same Lanczos filter for upscaling. But since any MLAA algorithm doesn't provide additional samples unlike TAA or MSAA, it seems that edge reconstruction is just a reverse process.